uber/tests/models/test_attendee.py

```diff
@@ -96,7 +96,7 @@ def test_badge():
     assert Attendee(ribbon=c.VOLUNTEER_RIBBON).badge == 'Unpaid Attendee (Volunteer)'
 
 
-def test_is_transferrable(monkeypatch):
+def test_is_transferable(monkeypatch):
     assert not Attendee(paid=c.HAS_PAID).is_transferable
```

Comment:
Ha.

I love how the "Referer" header is permanently misspelled and the misspelling is actually part of the official HTTP spec because some guy accidentally misspelled it like 20+ years ago and backwards-compatibility keeps us from ever fixing it.

Thankfully we're not at that point yet, though I sometimes wonder if we'll end up with that kind of "problem" if enough other events start using our code :) :( :/